projects/common/src/time/time-range.service.test.ts

```diff
@@ -88,4 +116,30 @@ describe('Time range service', () => {
       ['time']: new FixedTimeRange(new Date(1642296703000), new Date(1642396703000)).toUrlString()
     });
   });
+
+  test('Sets time range with navigation events', () => {
+    relativeTimeRange = new RelativeTimeRange(new TimeDuration(1, TimeUnit.Hour));
+    runFakeRxjs(({ cold, expectObservable }) => {
+      const spectator = buildService({
+        providers: [
+          mockProvider(NavigationService, {
+            get navigation$(): Observable<ActivatedRoute> {
```

Comment:
mockProvider(NavigationService, {
        navigation$: NEVER
      })

Reply:
This won't work i think, navigation$ needs to emit a value for the mock

Reply:
Oh my bad, when we spoke i thought you were referring to another instance of this pattern that was previously implemented. Fixed.